ffc-ahwr-mi-reporting/csv/create-filename.js

```diff
@@ -2,7 +2,7 @@ const month = ['January', 'February', 'March', 'April', 'May', 'June', 'July', '
 
 const createFilename = (filename) => {
   const createdDate = new Date()
-  return `${createdDate.getYear()}/${month[createdDate.getMonth()]}/${createdDate.getDay()}/${filename}`
+  return `${createdDate.getFullYear()}-${month[createdDate.getMonth()]}-${createdDate.getDay()}-${filename}`
```

Comment:
This format does not match the one in the sharepoint guide which is **YYYY MMDD Meaningful name Version**